app/javascript/mastodon/locales/ja.json

```diff
@@ -159,7 +159,7 @@
   "report.target": "{target} を通報する",
   "search.placeholder": "検索",
   "search_results.total": "{count, number}件の結果",
-  "standalone.public_title": "連合タイムライン",
+  "standalone.public_title": "今こんな話をしています",
```

Comment:
I was thinking about this translation 1 day.
but I can't find translation better than this.
and this translation is better than current one.
(new translation does not depend on what timeline(LTL/FTL) is shown)
So, I approve this.